pkg/webhook/add_handler.go

```diff
@@ -8,8 +8,9 @@ import (
 )
 
 func init() {
-	// AddToManagerFuncs is a list of functions to create webhook and add them to a manager.
-	AddToManagerFuncs = append(AddToManagerFuncs, fleetresourcehandler.Add)
+	// AddToManagerFleetResourceValidator is a function to register fleet guard rail resource validator to the webhook server
```

Comment:
Just a thought: we probably can refactor this a bit after resource-specific webhooks are no longer needed, but this can wait ;)